content/05.dl-complexities.md

```diff
@@ -1 +1,14 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
+
+Correctly training deep neural networks is a non-trivial process.
+There are many different options and potential pitfalls at every stage.
+To get good results you have to expect to train many networks with different tweaks and modifications.
```

Comment:
> "with different tweaks and modifications." 

 "tweaks and modifications" sounds good, but we should also say "settings" because changing things like the learning rate is not necessarily a tweak or modification (I think of tweaks and modifications more of larger changes, like adding dropout, batchnorm, varying the number of layers, and so forth)